tests/docker/Dockerfile

```diff
@@ -121,4 +121,8 @@ RUN mkdir -p /opt/scripts/consumer_offsets_recovery
 COPY --chown=0:0 tools/consumer_offsets_recovery /opt/scripts/consumer_offsets_recovery
 RUN python3 -m pip install --force --no-cache-dir -r  /opt/scripts/consumer_offsets_recovery/requirements.txt
 
+# Install remote_scripts
+RUN mkdir -p /opt/remote
+COPY --chown=0:0 tests/rptest/remote_scripts /opt/remote
+
```

Comment:
Maybe it would be better to use `inject_remote_script` method as other tests. This way we will not have to add remote scripts to CDT dependencies.

Reply:
There is also a readme in `remote_scripts`: https://github.com/redpanda-data/redpanda/blob/24200320930a6cc7e3c140e145f62ece393dfeca/tests/rptest/remote_scripts/README.md#L5

Reply:
we discussed it [before](https://github.com/redpanda-data/redpanda/pull/5238#discussion_r911880263), test specific files are injected with the injector, the common files go via Docker, in this case the scripts are gonna be reused with the other tests e.g. when 5238 is updated to use consumer groups and it's ready to go it also will depend on those files

Reply:
We need to remember to add this to CDT installer as it will always fail.